File: PYTHON/1045-tipos-de-triangulos.py
a, b, c = map(float, input().split(" "))

lista = [a, b, c]

i = 1
for i in range(len(lista)):
   for j in range(len(lista)-1):
      if lista[j] < lista[j+1]:
         lista[j], lista[j+1] = lista[j+1], lista[j]

a, b, c = lista[0], lista[1], lista[2]

if a >= b+c: print("NAO FORMA TRIANGULO")
elif pow(a,2) == pow(b,2)+pow(c,2): print("TRIANGULO RETANGULO")
elif pow(a,2) > pow(b,2)+pow(c,2): print("TRIANGULO OBTUSANGULO")
elif pow(a,2) < pow(b,2)+pow(c,2): print("TRIANGULO ACUTANGULO")

if a == b == c: print("TRIANGULO EQUILATERO")
elif (a==b) or (a==c) or (b==c): print("TRIANGULO ISOSCELES")

File: C/1045-tipos-de-triangulos.c
#include <stdio.h>
 
int main() {
 
    double A, B, C, aux;
    
    scanf("%lf %lf %lf",&A, &B, &C);
    
    while((A<B)||(B<C))
    {   
         if(A<B)
         {  aux = A; A = B; B = aux;  }
         if(B<C)
         {  aux = B; B = C; C = aux;  }
    }
    
    if(A>=B+C)
    {printf("NAO FORMA TRIANGULO\n");}
    
    else
    {
         if(pow(A,2)==pow(B,2)+pow(C,2))  
         {printf("TRIANGULO RETANGULO\n");} 
    
         else if(pow(A,2)>pow(B,2)+pow(C,2))
         {printf("TRIANGULO OBTUSANGULO\n");}
    
         else if(pow(A,2)<pow(B,2)+pow(C,2))
         {printf("TRIANGULO ACUTANGULO\n");}
    }
    
    if((A==B)&&(B==C))
         {printf("TRIANGULO EQUILATERO\n");}
    
    else if((A==B)||(B==C)||(A==C))
         {printf("TRIANGULO ISOSCELES\n");}
 
    return 0;
}